doc/source/whatsnew/v2.0.0.rst

```diff
@@ -794,6 +794,7 @@ Datetimelike
 - Bug in :func:`to_datetime` was raising ``ValueError`` when parsing empty string and non-ISO8601 format was passed. Now, empty strings will be parsed as :class:`NaT`, for compatibility with how is done for ISO8601 formats (:issue:`50251`)
 - Bug in :class:`Timestamp` was showing ``UserWarning``, which was not actionable by users, when parsing non-ISO8601 delimited date strings (:issue:`50232`)
 - Bug in :func:`to_datetime` was showing misleading ``ValueError`` when parsing dates with format containing ISO week directive and ISO weekday directive (:issue:`50308`)
+- Bug in :class:`DateOffset` when constructing with milliseconds in combination with another argument (:issue:`49897`)
```

Comment:
can we be more specific? like `Bug in :class:`DateOffset` was throwing ``TypeError```..., or whatever it was